net.damonlynch.rapid_photo_downloader.yaml

```diff
@@ -0,0 +1,153 @@
+app-id: net.damonlynch.rapid_photo_downloader
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+cleanup:
+  - "/include"
+  - /doc
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/vala
+  - "*.a"
+  - "*.la"
+  - "/lib/*.a"
+  - "/lib/*.la"
+  - "/lib/pkgconfig"
+  - "/include"
+  - "/lib/cmake"
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: rapid-photo-downloader
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+
+  #  For USB, until there is a better way in flatpak
+  - --device=all
+
+  # XDG commons folders
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+
+  # external disk mount
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --filesystem=/mnt
+
+  # notifications
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+  # file chooser
+  - --talk-name=org.freedesktop.portal.FileChooser
+
+  # App indicator ( not sure if needed)
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.Unity
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=org.ayatana.indicator.application
```

Comment:
Can you make sure ? There isn't a point in adding permissions if they aren't required, it opens up more sandbox holes.

Reply:
I remove them it seem not needed